src/components/pages/Host.jsx

```diff
@@ -168,14 +168,18 @@ class Host extends Component {
     }
 
     async advanceQuestionLocalAndServer() {
-        const {roomId,questionNum} = this.state;
+        const {roomId,questionNum,numDays} = this.state;
         const that = this;
         this.setState({
             questionNum: that.state.questionNum + 1,
         });
-        if (questionNum >= (await getNumDays(roomId)))
+        console.log("questionNum");
+        console.log(questionNum);
+        console.log("numDays");
+        console.log(numDays);
+        if (questionNum+2 >= numDays)
             this.setState({
-                phase: "ended",
+                phase: 'ended',
             })
```

Comment:
semi colon consistency
```suggestion
            });
```